parquet-column/src/main/java/org/apache/parquet/schema/LogicalTypeAnnotation.java

```diff
@@ -152,6 +162,10 @@ public String toString() {
     return sb.toString();
   }
 
+  public PrimitiveStringifier valueStringifier(PrimitiveType primitiveType) {
```

Comment:
`PrimitiveType.stringifier()` is for the API user. This one might be package protected.

Reply:
Will change the visibility to package protected too